### PR TITLE
Fix an issue where the $L(key,value) usage could not be parsed correctly

### PR DESCRIPTION
**Pull request by gouniLee**
Fix an issue where the $L(key,value) usage could not be parsed properly

### CODE REVIEW

File: JavaScriptFile.js [hunk 1]
```diff
@@ -264,13 +264,16 @@ JavaScriptFile.prototype.parse = function(data) {
...
             match = (result[8][0] === '"') ? result[9] : result[11];
+
+        } else if (result[7] === "key") {
+            key = (result[8][0] === '"') ? result[9] : result[11];
+            match = (result[2][0] === '"') ? result[3] : result[5];
         }
 
         if (match && match.length) {
-            this.logger.trace("Found string key: " + this.makeKey(match) + ", string: '" + match + "'");
+            this.logger.trace("Found string key: " + this.makeKey(key) + ", string: '" + match + "'");
```
Comment (ehoogerbeets) [inline JavaScriptFile.js]: This is only logger output, so not so important, but the key should be just "key" instead of `this.makeKey(key)` because it is an explicit key from the developer, not a generated key.

File: README.md [hunk 2]
```diff
@@ -5,9 +5,10 @@ allows it to read and localize javascript files. This plugins is optimized for w
 
 ## Release Notes
 v1.4.6
-* updated dependencies
+* Update dependencies
 * Use the logger provided by the loctool instead of using log4js directly.
 * Add node 16 version testing for circleCI (minimum version of node is v10)
+* Fixe an issue where the $L(key,value) usage could not be parsed properly.
```
Comment (ehoogerbeets) [inline README.md]: Fixe -> Fix

**Review by ehoogerbeets — APPROVED** [review]
My comments were minor, so I approved this anyways.

**Comment by gouniLee** [comment]
> My comments were minor, so I approved this anyways.
Thanks. I updated them.
